packages/tests/esm/migrations/1580658856631_author.cjs

```diff
@@ -1,9 +1,18 @@
+const { createEnumTable, enumArrayColumn } = require("joist-migration-utils");
+
 exports.up = (b) => {
+  createEnumTable(b, "color", [
+    ["RED", "Red"],
+    ["GREEN", "Green"],
+    ["BLUE", "Blue"],
+  ]);
```

Comment:
Adding an enum to test that logic here (that's what exposed the need for a fix in the joist-codegen package).